README.md

```diff
@@ -1,3 +1,601 @@
+
+#### An example of shortened license text
+ ```java
+/*
+ * Copyright (C) 2021 Arthur Aukhatov
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+ ```
+
+### Import Statements
+Dependencies in *import statements* should be structured in the following way:
+
+* import `java.*`
+* `<blank line>`
+* import `javax.*`
+* `<blank line>`
+* 3rd party imports
+* `<blank line>`
+* import `your.own.imports`
+* `<blank line>`
+* other static imports
+
+> It's not recommended to use *wildcard*s - if there are too many imports it could mean that you should split the
+> class into several classes.
+
+### Java Class Structure
+The order of the `java` class elements: 
+
+1. **static** class fields
+2. **public** instance fields
+3. **protected** instance fields
+4. **private** instance fields
+5. **constructor without parameters**
+6. **other constructors**
+7. **private** methods, which are called from a constructor
+8. **static** fabric methods
+9. **getters**, **setters**
+10. interface methods implementations
+11. **private** or **protected** methods, called from a method with the interface method implementation
+12. other methods
+13. `equals`, `hashCode`, `toString`
+
+### Formatting
+#### Curly Braces
+Braces layout should follow [Kernighan and Ritcle (a.k.a. "Egyptian brackets")](https://en.wikipedia.org/wiki/Indentation_style#K.26R):
+* No line breaks before the opening brace
+* One space should precede opening brace
+* A line break should immediately follow opening brace and closing brace, except for multi-block expressions like `if/else` and `try/catch/finally`
+
+##### try/catch example
+Correct:
+```java
+try {
+    file = openFile("custom_file.txt");
+} catch (FileNotFoundException e) {
+    file = defaultFile();
+}
+```
+
+Incorrect:
+```java
+try
+{
+    file = openFile("custom_file.txt");
+}
+catch (FileNotFoundException e)
+{
+    file = defaultFile();
+}
+```
+
+##### if/else example
+Correct:
+```java
+if (flag == true) {
+    value = 1000;
+} else {
+    value = 1200);
+}
+```
+
+Incorrect:
+```java
+
+if (flag == true){
+    value = 1000;
+}else{value = 1200);}
+
+```
+
+##### Class definition example
+Correct:
+```java
+public class Cat implements Animal {
+
+    @Override
+    protected void speak() {
+        this.meow();
+    }
+}
+```
+
+Incorrect:
+```java
+public class Cat implements Animal
+{
+    @Override
+    protected void speak()
+    {
+        this.meow();
+    }
+}
+```
+
+#### Line breaks with operators and expressions
+If the expression's length exceeds **120 symbols**, delimiters (commas, operators, etc.) should be left at the end of the line, 
+and not be moved to the next line. Exception - in the method call, like *Builder Pattern*, *Java Streams*.
+In lambda expressions `-> {` should be left on the first line.
+
+##### if example
+Correct:
+```java
+if (veryImportantFlagWithLongName == true && theLineIsLongerThanShouldBe == true &&
```

Comment:
I've changed my mind. Vice versa, if you don't mind. I mean with the leading operator.

Reply:
Ok, thought about it too :)

Reply:
done